### PR TITLE
Cleanup of README

### PR DESCRIPTION
**Pull request by zkamvar**
I removed the table linking to old md documents and added instructions for installing dependencies to run vignettes.

@hlapp, what do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,20 +2,17 @@
...
-This site provides resources for conducting population genetic analyses in R using existing packages. 
+## Introduction
 
-| [Get started](GETSTARTED.md) | [Learn](LEARN.md)                         |
-| :--------------:|:-------------------------:        |
-| [**Using R**](USINGR.md)     | [**Developer**](DEVELOPER.md) |
-
-
-Any member of the R user community proficient in population genetics is welcome to contibute to this site. See [here](CONTRIBUTING.md) for instructions on how to contribute workflow vignettes using [R markdown](R_MARKDOWN.md).
+This site provides resources for conducting population genetic analyses in R using existing packages. Any member of the R user community proficient in population genetics is welcome to contibute to this site. See [here](CONTRIBUTING.md) for instructions on how to contribute workflow vignettes using [R markdown](R_MARKDOWN.md).
```
Comment (hlapp) [inline README.md]: Why not just `See [instructions](CONTRIBUTING.md) on how to ...`?

File: README.md [hunk 2]
```diff
@@ -2,20 +2,17 @@
...
 
 This site was developed during the [Population Genetics in R hackathon]
```
Comment (hlapp) [inline README.md]: In the course of cleanup, perhaps change this to _Development of this site started at ..._?

**Comment by hlapp** [comment]
Only two minor comments. Otherwise looks great!

File: README.md (hunk 1 above)
Comment (zkamvar) [inline README.md]: > Why not just See [instructions](CONTRIBUTING.md) on how to ...?

:+1: 

Good catch. I'm always a fan of reducing the overuse of `[here](link_that_could_go_literally_anywhere)`.

File: README.md (hunk 2 above)
Comment (zkamvar) [inline README.md]: :+1:

**Comment by zkamvar** [comment]
Changes have been made. You may pull when ready.

**Pull request merged**